recipes/newrelic/infrastructure/ohi/kafka/linux.yml

```diff
@@ -0,0 +1,50 @@
+# Visit our schema definition for additional information on this file format
+# https://github.com/newrelic/open-install-library/blob/main/docs/recipe-spec/recipe-spec.md#schema-definition
+
+name: kafka-open-source-integration
+displayName: kafka Open Source Integration
+description: New Relic install recipe for default kafka Open Source on-host integration (via Infra-Agent)
+repository: https://github.com/newrelic/nri-kafka
+
+dependencies:
+  - infrastructure-agent-installer
+
+installTargets:
+  - type: host
+    os: linux
+    kernelArch: x86_64
+
+keywords:
+  - Infrastructure
+  - Integration
+  - kafka
+
+processMatch:
+  - kafka
```

Comment:
@NRhzhao this is working, however I wonder if we should use the same regex pattern than we use on windows "java.*kafka" to make it a bit more specific to that java process when detecting?

Reply:
@Julien4218 Will update.